doc/content/getting-started/migrating/migrating-from-v2/_index.md

```diff
@@ -55,3 +55,8 @@ There are two approaches for migrating devices, depending on how many end device
 
 - [Using {{% tts %}} Console]({{< relref "migrate-using-console" >}}) - this method is convenient only for small number of devices. Migrating active device sessions is not supported.
 - [Using `ttn-lw-migrate` tool]({{< relref "migrate-using-migration-tool" >}}) - this method allows migrating devices in bulk, optionally with active device sessions.
+
+## Migration decision tree
+Migrating from V2 to The Things Stack Cloud or The Things Stack Community Edition (click on the image to enlarge).
```

Comment:
When I look at this diagram again, I see a bit of gray area. We ask "Are you using TTN V2?" but we are not asking "Are you migrating to TTS Community Edition?". Theoretically, people could be moving from TTN V2 to any TTS deployment, doesn't have to be Community Edition, right?

For example, if someone wants to migrate from TTN V2 to TTS Enterprise, TTS Enterprise has to be configured manually to connect to Packet Broker in order for migration of end device to work without migrating a gateway. But we don't ask "Is your The Things Stack deployment connected to Packet Broker?". 

On the other hand, if we keep expanding this diagram to be 100% correct, it will become too confusing again. It might be a good idea to break this into 2 separate diagrams, one for TTN V2 and another for TTI V2.. What do you think?

Reply:
It does say at the beginning: 
> Migrating from V2 to The Things Stack Cloud or The Things Stack Community Edition. 

Let me stress that a bit by making The Things Stack Cloud and The Things Stack Community Edition bold. 

I'd suggest to merge this for now, I can have another look at improving this visual later on.